app/models/task.py

```diff
@@ -3,4 +3,8 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True);
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    
```

Comment:
add goal_id with a foreign key `goal.goal_id` to finish the connection between the models